src/js/structs/Job.js

```diff
@@ -102,7 +102,13 @@ module.exports = class Job extends Item {
   }
 
   getSchedules() {
-    return this.get('schedules') || [];
+    const schedules = this.get('schedules');
+
+    if (!Array.isArray(schedules)) {
+      return [];
```

Comment:
Let's create a ticket to track both 
1. documentation that should be fixed
2. raml validation that should be added to jobs